server/translations.go

```diff
@@ -25,3 +29,83 @@ func (s *Server) handleTranslationPR(ctx context.Context, pr *model.PullRequest)
 		return
 	}
 }
+
+func (s *Server) handleModificationOfLanguageFiles(ctx context.Context, pr *model.PullRequest) {
+	repoConfig, isTranslated := isTranslatedRepo(pr, s.Config.TranslationsRepos)
+	if !isTranslated {
+		return
+	}
+
+	if s.Config.TranslationsBot == pr.Username {
+		return
+	}
+
+	prFiles, err := s.getPRFiles(ctx, pr.RepoOwner, pr.RepoName, pr.Number)
+	if err != nil {
+		mlog.Error("Error listing PR files", mlog.String("repo", pr.RepoName), mlog.Int("pr", pr.Number), mlog.Err(err))
```

Comment:
This is okay for this PR. But in future PRs, it's better to propagate the error upwards.